### ISSUE TITLE
Add `bootstrap` command to `neo-one-server-plugin-network`

### ISSUE THREAD

**Issue by dicarlo2** [neo-one-suite/neo-one#80]
This command should bootstrap a private network node with a bunch of data. To start, let's bootstrap with
 - ContractTransactions (`client.transfer`)
 - Register a couple of assets (`client.registerAsset`)

Basically you'll want to add the `master` wallet that's automatically created with the private network to the `client` along with creating some other wallets to transfer and register assets with. That should be relatively straightforward for the first PR. I actually started on this, here's a gist of a WIP: https://gist.github.com/dicarlo2/4b6f3234c95443a570786828a38508d2. You'll just need to add it as an interactive command to the network plugin and then fill in the file.

Next, we'll want to add more transactions:
 - ClaimTransactions (`client.claim`)
 - Issue coins for the assets created above (`client.issue`)

However, this requires the previous transactions to be confirmed in a block. And waiting for a block to be created every time we need one will make this command take a long time. So, before adding the above transactions, let's add a new client type to `neo-one-client`: `DeveloperClient`. This client should work similar to `ReadClient` in that it talks to a single network (unlike `Client` which can talk to many networks) and takes a new `DeveloperProvider` type. It should have the following methods (which basically just delegate to `DeveloperProvider`:
 - `startConsensus()`: triggers consensus on the node immediately
 - `updateOptions(options: Options)`: updates various options for the node. Initially, it will just have one option `secondsPerBlock` which should update the `blockchain.settings.secondsPerBlock`. Since we're all about immutability, you'll probably want to change `settings` in `Blockchain` to be a `BehaviorSubject` and add a method `get settings()` which just takes the current value so it's backwards compatible. That way you can update it.
 - `fastForward(seconds: number)`: fast forward the block time by `seconds` seconds. Take a look at `runConsensus` to see where you need to feed this value in order to force the next block to use a time in the future.

Each method should probably be it's own PR since they're relatively independent and will require a decent amount of work to thread the values from `neo-one-client`, through `neo-one-node-rpc` to their final destination. For the `bootstrap` command we'll likely only use `startConsensus`, but the other 2 will come in handy later. Remember to add tests so we keep `neo-one-client` at near 100% coverage!

Once that's done, we can easily add the other transactions mentioned above in another PR. Spread the transactions out over a few blocks using `startConsensus` so that we have a good number of blocks and transactions.

Finally, let's add a few NEP-5 contracts:
 - Publish a few NEP-5 contracts with various parameters using `client.publish`
 - Invoke their `deploy` methods so that they're setup properly (using a smart contract method on `client.smartContract`)
 - Fast forward time to when the ICO begins and call `mintTokens` with some addresses (same as above, use a smart contract method)
 - Transfer the minted tokens to various addresses

This can all be done in one PR.

**Comment by dicarlo2** [comment]
This is done except for invoking smart contracts. Opening up a followup to do that once the TypeScript compiler is complete.
